testSchema/imageUrl/index.ts

```diff
@@ -0,0 +1,13 @@
+import { list } from "@keystone-6/core"
+import { allowAll } from "@keystone-6/core/access"
+import { Lists } from ".keystone/types"
+import { imageUrl } from "../../src/url/imageUrl"
+
+export const ImageUrl: Lists.Url = list({
+  access: allowAll,
+  fields: {
+    linkToBinouImage: imageUrl({
```

Comment:
it's about time I reveal to you that it's [biniou](https://www.ericdentinger.com/fr/biniou.html) :-p